src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -32,7 +32,7 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol
     /// Exporter consuming <see cref="Activity"/> and exporting the data using
     /// the OpenTelemetry protocol (OTLP).
     /// </summary>
-    public class OtlpExporter : BaseExporter<Activity>
+    internal class OtlpExporter : BaseExporter<Activity>
```

Comment:
the ctor() was already internal, so we wont break anyone right?

Reply:
By the class being public, the type is visible, so a user can type `var exporter = new OtlpExporter()` ... but with no way to construct it, is there any value in exposing it?

Reply:
Agree.
I was thinking -- if a user want to write own Batching/SimpleExportprocessor, then they wont be able to do that, as they cannot construct the exporter. But its okay, as spec talks about only 2 ExportingProcessors. and https://github.com/open-telemetry/opentelemetry-dotnet/pull/1504 will solve customization issues as well.

Reply:
Right, if #1504 is the pattern we want going forward, then I don't think we need to expose the classes. We could always re-expose the classes with a public constructor if we saw some need for that in the future.